src/main/java/sg/ncl/ConnectionProperties.java

```diff
@@ -429,4 +429,26 @@ public String getUsageStat(String teamId, String... paramString) {
         }
         return HTTP_MODE + sioAddress + ":" + sioPort + "/" + analyticsEndpoint + "/usage/teams/" + teamId + params.toString();
     }
+
+    //-------------------------------------
+    // RESERVATION
+    //-------------------------------------
+    public String getReservationStatus(String teamId) {
+        return HTTP_MODE + sioAddress + ":" + sioPort + "/" + teamEndpoint + "/" + teamId + "/reservations";
+    }
+
+    public String releaseNodes(String teamId, Integer numNodes) {
+        return HTTP_MODE + sioAddress + ":" + sioPort + "/" + teamEndpoint + "/" + teamId + "/reservations?numNodes=" + numNodes;
+
+    }
+
+    public String reserveNodes(String teamId, Integer numNodes, String machineType) {
+        if (machineType == null) {
+            // HTTP_MODE + sioAddress + ":" + sioPort + "/" + teamEndpoint + "/" + teamId + "/reservations?numNodes=" + numNodes
+            return releaseNodes(teamId, numNodes);
+        } else {
+            // HTTP_MODE + sioAddress + ":" + sioPort + "/" + teamEndpoint + "/" + teamId + "/reservations?numNodes=" + numNodes + "&machineType=" + machineType;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)